react-components/src/components/CompareMarkets/DataTable.jsx

```diff
@@ -169,7 +178,11 @@ export default function DataTable(props) {
     )
   }
 
-  setBaseYear(cache[datasetName], comparisonMarkets, config)
+  const addRemoveShortlist = (market, add) => {
+    ;[addMarketItem, removeMarketItem][add ? 0 : 1](market)
```

Comment:
Not sure this is very readable, would this work?

```suggestion
    (add ? addMarketItem : removeMarketItem)(market)
```

Or maybe the slightly more verbose, but definitely clearer:
```suggestion
    add ? addMarketItem(market) : removeMarketItem(market)
```